packages/scandipwa/src/component/CurrencySwitcher/CurrencySwitcher.style.scss

```diff
@@ -73,6 +73,10 @@
                 }
             }
 
+            &-OptionsWrapper_isExpanded {
+                scrollbar-width: thin;
```

Comment:
This option is unsupported by most browsers and definitely isn't the fix for this issue.
Did you manged to reproduce it?

Reply:
I'd say we have multiple wrong styles here that are merged into a working result.
Please take a look at the `.CurrencySwitcher .FieldSelect-Options`.
It has the `overflow-y: auto` property, but the `max-height` is `fit-content`.
Next we have the `.FieldSelect-OptionsWrapper_isExpanded` with the `overflow-y: auto` and fixed `max-height` value.
I'd suggest to remove the `overflow` property from the first element.
Next to set `overflow-x: hidden` to the second one in the `CurrencySwitcher` style file.